README.md

```diff
@@ -892,19 +893,31 @@ Connecting to main.ton.dev
 Running get-method...
 Succeeded.
 Result: ["125387107580525"]
+
+$ tonos-cli runget --tvc acc.tvc compute_returned_stake 0x0166d0181a19f87af9397040a68671e1b239f12152824f7d987fd6897d6a9587
+Config: /home/user/TONLabs/tonos-cli/tonos-cli.conf.json
+Input arguments:
+ address: acc.boc
+  method: compute_returned_stake
+  params: ["0x0166d0181a19f87af9397040a68671e1b239f12152824f7d987fd6897d6a9587"]
+Connecting to main.ton.dev
+Running get-method...
+Succeeded.
+Result: ["125387107580525"]
 ```
 
 
 
 ### 4.4.5. Run contract method locally for saved account state
 
 ```bash
-tonos-cli run --boc [--abi <contract.abi.json>] <account> <method> <params> [--bc_config <config_path>]
+tonos-cli run [--boc] [--tvc] [--abi <contract.abi.json>] <account> <method> <params> [--bc_config <config_path>]
```

Comment:
I realized that `--boc` is not a very good name. Butt now we should use it for compatibility.